policies/lib/tests/integration/user_absent_test.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2025 Normation SAS
+
+use crate::integration::{end_test, get_lib_path, init_test};
+use crate::testlib::method_test_suite::MethodTestSuite;
+use crate::testlib::method_to_test::{method, MethodStatus};
+use users::{get_user_by_name};
+
+#[test]
```

Comment:
I would prefer to tag the test as ignored by default as we want to avoid to define a toto user on every dev workstation that will run the tests